### ISSUE TITLE
Anyway to completely customise the events title

### ISSUE THREAD

**Issue by AryaaSk** [Quivr/iOS-Week-View#30]
when you give the events title and the events date the event automatically gets the title with the name and the title and || with the location data next to it, is there a way to have complete control of the text on an event

**Comment by reilem** [comment]
Yes you should be able to overwrite the `EventData` class and implement your own `getDisplayString`. Sorry for the late reply!
